src/drawing/DrawingList.tsx

```diff
@@ -40,6 +41,17 @@ export function DrawingList({ activeId = null, annotations, className, onSelect
     });
     useOutsideEvent('mouseup', rootElRef, (): void => setIsListening(true));
 
+    React.useEffect(() => {
```

Comment:
Storing the dimensions in local state presents challenges with keeping the state updated, such as when the window is resized. If we can't work around this via SVG and/or CSS properties, we should consider passing down a function that returns the `rootElRef` current value. Something like this:

```
export function DrawingList({ activeId = null, annotations, className, onSelect = noop }: Props): JSX.Element {
    const rootElRef = React.createRef<SVGSVGElement>();
    const getRootEl = () => rootElRef.current;

    // ... later...

    <DrawingTarget getParentEl={getRootEl} />
};